src/Faker/Calculator/TCNo.php

```diff
@@ -14,39 +16,24 @@ class TCNo
      *
      * @param string $identityPrefix
      * @return string Checksum (two digit)
+     * @deprecated Use {@link \Faker\Provider\tr_TR\Person::tcNoChecksum()} instead
+     * @see \Faker\Provider\tr_TR\Person::tcNoChecksum()
      */
     public static function checksum($identityPrefix)
     {
-        if (strlen((string) $identityPrefix) !== 9) {
-            throw new InvalidArgumentException('Argument should be an integer and should be 9 digits.');
-        }
-
-        $oddSum = 0;
-        $evenSum = 0;
-
-        $identityArray = array_map('intval', str_split($identityPrefix)); // Creates array from int
-        foreach ($identityArray as $index => $digit) {
-            if ($index % 2 == 0) {
-                $evenSum += $digit;
-            } else {
-                $oddSum += $digit;
-            }
-        }
-
-        $tenthDigit = (7 * $evenSum - $oddSum) % 10;
-        $eleventhDigit = ($evenSum + $oddSum + $tenthDigit) % 10;
-
-        return $tenthDigit . $eleventhDigit;
+        return \Faker\Provider\tr_TR\Person::tcNoChecksum($identityPrefix);
     }
 
     /**
      * Checks whether a TCNo has a valid checksum
      *
      * @param string $tcNo
      * @return bool
+     * @deprecated Use {@link \Faker\Provider\tr_TR\Person::tcNoIsValid()} instead
```

Comment:
```suggestion
     * @deprecated use {@link \Faker\Provider\tr_TR\Person::tcNoIsValid()} instead
```

Reply:
Okay, then we probably want to change the ones in 

https://github.com/FakerPHP/Faker/blob/2e1bc4c64cdf8cb101cc416b4a660129af6029f5/src/Faker/Provider/Barcode.php#L39
https://github.com/FakerPHP/Faker/blob/2e1bc4c64cdf8cb101cc416b4a660129af6029f5/src/Faker/Provider/Barcode.php#L24
https://github.com/FakerPHP/Faker/blob/2e1bc4c64cdf8cb101cc416b4a660129af6029f5/src/Faker/Provider/Image.php#L15

etc. also ;)